### DISCUSSION TITLE
tauonmusicbox.rocks mobile version

### DISCUSSION THREAD

**Discussion by Solarunit** [Taiko2k/Tauon#429] [General]
Site looks pretty bad on smartphone.

**Comment by Taiko2k** [comment]
Ehh its not a mobile app so its not too much of a concern I don't think. The site could probably use a redesign eventually and perhaps that could be a consideration then. For now its the best I can do.

**Comment by da2x** [comment]
Here you go: [tauonmb-site pull request #1: Mobile support](https://github.com/Taiko2k/tauonmb-site/pull/1)

**Reply by Taiko2k** [reply]
That's much better on mobile. Thanks!

**Reply by da2x** [reply]
It’s not perfect but it gets the job done without excess scrolling. ^__^

**Comment by Solarunit** [comment]
There is a problem: when you swipe to the left you can see an empty space on the right side,
Everything else is fine.

**Reply by da2x** [reply]
Fixed in [pull #2](https://github.com/Taiko2k/tauonmb-site/pull/2).

**Reply by Solarunit** [reply]
Nice! Thanks!
